.github/workflows/main.yml

```diff
@@ -109,7 +109,7 @@ jobs:
   # MANDATORY TESTING USING LOWEST SUPPORTED TOOLCHAIN
   lowest_supported:
     env:
-      TOOLCHAIN: 1.74.0  # LOWEST SUPPORTED RUST TOOLCHAIN
+      TOOLCHAIN: 1.77.0  # LOWEST SUPPORTED RUST TOOLCHAIN
```

Comment:
You can drop the toolchain bumps, they are not necessary now that bindgen rust target is pinned.